plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/types/ClassEmbeddingDetails.kt

```diff
@@ -123,4 +129,15 @@ class ClassEmbeddingDetails(val type: ClassTypeEmbedding, val isInterface: Boole
             } ?: listOf()
         }
     }
+}
+
+class StringEmbeddingDetails(type: ClassTypeEmbedding) : ClassEmbeddingDetails(type, false) {
+
```

Comment:
nit: imho starting a class with an empty line like this looks sloppy, does the autoformatter not remove it?